Machine_Learning/notebooks/czmodel/Regresssion_PyTorch_5_0_0.ipynb

```diff
@@ -0,0 +1,669 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "cHZX_QBJAsxR",
+    "outputId": "81d368f6-4b14-479c-cf2f-12574f460fe9",
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Install czmodel and dependencies\n",
+    "! pip install --upgrade pip\n",
+    "! pip install \"czmodel[pytorch]>=5.0,<6.0\""
```

Comment:
Do you actually have to specify the minor version here? IMO czmodel[pytorch]>=5,<6 should do.